src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.WebUtilities.Encoders
```

Comment:
in `Microsoft.Framework.DependencyInjection` namespace like rest other `IServiceCollection` extensions?